packages/cli/src/commands/dbCommands/generate.js

```diff
@@ -21,6 +21,12 @@ export const builder = (yargs) => {
       description: 'Print more',
       type: 'boolean',
     })
+    .option('schema', {
+      alias: 's',
```

Comment:
```suggestion
```
I would rather leave the alias out for now

Reply:
The reason why I suggest we leave it out for now is that we might want to use `s` for the upcoming migration changes.